src/utils/querySelectorHelper.js

```diff
@@ -121,12 +126,29 @@ function matchRule(element, rule) {
     }
 
     if (rule.hasOwnProperty('attrs')) {
-        if (!rule.attrs.every(attr => {
+        if (!rule.attrs.some(attr => {
+            if (!element.hasAttribute(attr.name)) {
+                return false;
+            }
+
+            let value = element.getAttribute(attr.name);
```

Comment:
if you want to set value here, you could remove undefined from the switch and add a if before:

```js
if (attr.operator === undefined) {
    return element.hasAttribute(attr.name);
}
```

Because for this case we don't need the value